### DISCUSSION TITLE
blob not null?

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-query#562] [Q&A]
How would I write this without using extra?
```
"filter" blob NOT NULL
.col(
    ColumnDef::new(SavedFilters::Filter)
        .extra("blob NOT NULL".to_string()),
)
```

**Comment by billy1624** [comment]
Hey @JediMaster25, welcome! First, I'd like to know which database are you using?

**Reply by ghost** [reply]
It doesn't matter the output should work for all supported engines right? Anyways I'm using SQLite.

**Reply by ikrivosheev** [reply]
@JediMaster25, hello, try something like this:

`ColumnDef::new_with_type(SavedFilters::Filter, ColumnType::Binary(BlobSize::Tiny)).not_null()`
